### ISSUE TITLE
Warning and error with 

### ISSUE THREAD

**Issue by vtashkov** [dmvvilela/svelte-scrollactive#5]
I am getting this warning on the start of my project:
`WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.

svelte-scrollactive@0.0.9

Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.`

And then this error:
`Unknown file extension ".svelte" for ../node_modules/svelte-scrollactive/scrollactive.svelte
at Object.getFileProtocolModuleFormat [as file:] (node:internal/modules/esm/get_format:217:9)
    at defaultGetFormat (node:internal/modules/esm/get_format:243:36)
    at defaultLoad (node:internal/modules/esm/load:123:22)
    at async ModuleLoader.load (node:internal/modules/esm/loader:567:7)
    at async ModuleLoader.moduleProvider (node:internal/modules/esm/loader:442:45) {
  code: 'ERR_UNKNOWN_FILE_EXTENSION'
`

I am using Svelte 4.2.19. Maybe it isn't updated?

**Comment by dmvvilela** [comment]
Yeah, we might need to update this package. Care to send a PR? @vtashkov

**Comment by vtashkov** [comment]
> Yeah, we might need to update this package. Care to send a PR? @vtashkov

I don't have any, I am sorry. I went to develop my own ad-hoc solution.
